benchmark/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "benchmark",
-  "version": "6.0.0-alpha.0",
+  "version": "6.0.0",
```

Comment:
Mistake or you want to avoid updating this non-releasing package every release? 🙈